requirements.txt

```diff
@@ -1,9 +1,9 @@
 dj-database-url==0.3.0
 Django==1.8.13
-django-debug-toolbar==1.3.2
 django-jinja==1.4.1
 django-jinja-bootstrap-form==4.0.3
 funcy==1.7.2
+django-model-utils==1.3.1
```

Comment:
It install django-model-utils-2.6
At 2.4 PassThroughManager was removed from django-model-utils  https://github.com/carljm/django-model-utils/blob/68bc61e82517cb01fa36c0d007091a8670425913/CHANGES.rst#24-20151203

So it should be 
`django-model-utils>=1.4,<2.4`